index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- Project Header -->
+    <header>
+        <!-- Header toolbar -->
+        <nav>
+            <div class="content header-toolbar-container">
+                <div class="logo-container">
+                    <div class="logo-item">
+                        <h5 class="logo">My Logo</h5>
```

Comment:
- [ ] Replace h5 with a link tag. The logo the most of the time is a link that redirects to the homepage in this case the intro section.